Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -53,8 +53,9 @@ inline CGPoint __CreateCGPointWithTransform(CGFloat x, CGFloat y, const CGAffine
     bool figureClosed;
     CGPoint currentPoint{ 0, 0 };
     CGPoint startingPoint{ 0, 0 };
+    const CGAffineTransform* lastTransform;
```

Comment:
Storing this as a pointer is bad news: you'll probably be keeping a pointer to a transform that was on the user's stack.